src/test/kotlin/io/github/rybalkinsd/kohttp/interceptors/RetryInterceptorTest.kt

```diff
@@ -0,0 +1,118 @@
+package io.github.rybalkinsd.kohttp.interceptors
+
+import io.github.rybalkinsd.kohttp.client.defaultHttpClient
+import io.github.rybalkinsd.kohttp.client.fork
+import io.github.rybalkinsd.kohttp.dsl.httpGet
+import io.mockk.spyk
+import io.mockk.verify
+import okhttp3.OkHttpClient
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.mockserver.client.MockServerClient
+import org.mockserver.integration.ClientAndServer
+import org.mockserver.integration.ClientAndServer.startClientAndServer
+import org.mockserver.matchers.Times.exactly
+import org.mockserver.model.HttpRequest.request
+import org.mockserver.model.HttpResponse.response
+import java.net.SocketTimeoutException
+import java.util.concurrent.TimeUnit
+
+
+class RetryInterceptorTest {
+
+    private lateinit var mockServer: ClientAndServer
+    private val mockServerPort: Int = 1080
```

Comment:
m/b we can omit port in these tests. (or use default 80)

Reply:
it is better to set port